README.md

```diff
@@ -48,12 +55,31 @@ nctl-view-user-secret-key user=3
 
 ### Run the container with predefined account keys
 
-Each time the container is started, nctl runs with a set of randomly generated account keys. To use a set of predefined and pregenerated account keys, run the container with an extra parameter:
+Each time the container is started, nctl runs with a set of randomly generated account keys. To use a set of predefined and pregenerated account keys, run the container with the environment variable `PREDEFINED_ACCOUNTS` set to `true`:
 
 ```bash
-docker run --rm -it --name mynctl -d -p 11101:11101 makesoftware/casper-nctl /bin/bash -c "source /home/casper/restart-with-predefined-accounts.sh"
+docker run --rm -it --name mynctl -d -p 11101:11101 --env PREDEFINED_ACCOUNTS=true makesoftware/casper-nctl
 ```
 
+### Change the settings using environment variables
+
+Using environment variables you can tweak few parameters to change the frequency of blocks and the  deploys processing speed.
+
+| Environment variable   | Default value | Description                                                                                                                                                                                                                                                                           |
+|------------------------|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| MINIMUM_ROUND_EXPONENT | 12            | Integer between 0 and 255.  The power of two that is the number of milliseconds in the minimum round length, and therefore the minimum delay between a block and its child.  E.g. 12 means 2^12 milliseconds, i.e. about 4 seconds.                                                   |
```

Comment:
Are 0 and 255 included? If so, making it explicitly known might be useful.

Reply:
Fixed. 0 is included, 255 is not.
Added also a warning note. Too low or too high values will lead to nodes malfunction even allowed.